src/utils/sampler_metrics.ts

```diff
@@ -17,6 +17,12 @@ const SAMPLER_BLOCK_NUMBER_GUAGE = new Gauge({
     help: 'Provides information about the gas limit detected during a sampler call',
```

Comment:
Is there a reason for the const to be named `SAMPLER_BLOCK_NUMBER_GUAGE` instead of "SAMPLER_BLOCK_NUMBER_GAUGE"?

Reply:
probably no reason other than fat fingers 😂 , also mine should probably not prefixed with `SAMPLER_` 🙈